src/daemon/plugin.c

```diff
@@ -2234,8 +2239,35 @@ void plugin_log(int level, const char *format, ...) {
 
     le = le->next;
   }
+} /* void _daemon_log */
+
+void plugin_log(int level, const char *format, ...) {
+  char msg[1024] = "";
+
+  char const *name = plugin_get_ctx().name;
+  if (name != NULL)
+    snprintf(msg, sizeof(msg), "%s plugin: ", name);
```

Comment:
please check for error / truncation.

Reply:
What should we do on error ?

Reply:
either pass it up to the caller, or just log it.